python/src/scipp/show.py

```diff
@@ -106,6 +105,13 @@ def _extents(self):
                 e.append(1)
         return [1] * (3 - len(e)) + e
 
+    def _events_height(self):
+        if self._variable.events is None:
+            return 0
+        events = self._variable.events
+        # Rough estimate of vertical space taken by depiction of events buffer
+        return 1 + 1.3 * (len(events.meta) + len(events.masks))
```

Comment:
Looks like there is a bug here when events is type `Variable`.

Reply:
Good catch, thanks.